<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WellPlotStyleWidget</class>
 <widget class="QWidget" name="WellPlotStyleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="wellPlotStyleOuterWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="QWidget" name="wellPlotStyleWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QWidget" name="plotTitleWidget" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0" colspan="2">
             <widget class="QCheckBox" name="plotTitleOnCheckBox">
              <property name="text">
               <string>Plot title displayed</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="plotTitleLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Plot title</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="plotTitleLineEdit">
              <property name="maximumSize">
               <size>
                <width>276</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QLabel" name="plotBackgroundOpacityLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Track background opacity</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="plotBackgroundColorLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Track background colour</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="trackBackgroundOpacitySpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QWidget" name="trackBGColourBtnHolderWidget" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="headerLabelWidget" native="true">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QCheckBox" name="singleRowLabelsCheckBox">
              <property name="toolTip">
               <string>Fit labels to a single row</string>
              </property>
              <property name="text">
               <string>Single row header labels</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelBackgroundColorLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Header label background colour</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QSpinBox" name="labelForegroundOpacitySpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelBackgroundOpacityLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Header label background opacity</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSpinBox" name="labelBackgroundOpacitySpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelForegroundColorLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Header label text colour</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelForegroundOpacityLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Header label text opacity</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="headerLabelHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <widget class="QWidget" name="headerLabelBGColourBtnHolderWidget" native="true"/>
            </item>
            <item row="3" column="2">
             <widget class="QWidget" name="headerLabelTextColourBtnHolderWidget" native="true"/>
            </item>
            <item row="3" column="1">
             <spacer name="headerTextHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="plotVerticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>136</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
